src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -2865,6 +2978,23 @@ public bool Contains(TKey key)
                 } while (g != _lastGrouping);
             }
         }
+        
+        public IGrouping<TKey, TElement>[] ToArray()
```

Comment:
While you're at it, I know this is nitpicky, but would you mind changing "behaviour" -> "behavior" (similar to the "optimize" change)?

Also, one other nit I noticed that was hard to see in the diff: this PR introduces a bunch of trailing whitespace that could be removed (tab spaces on "blank lines").